scipy-1.0/paper.tex

```diff
@@ -263,6 +263,26 @@ \subsection*{Numerical optimization}
 
 
 \subsection*{Statistical distributions}
+The \texttt{scipy.stats} module provides a framework to implement probability
+distributions.  Given an expression for a probability distribution function (pdf),
+the framework will calculate moments, cumulative distribution function (cdf),
+differential entropy, and more. Those properties are calculated based on
+generic code, e.g., numerical integration of the pdf to obtain the cdf. In cases
```

Comment:
I would replace the sentence

> In cases where that generic code does not provide sufficient accuracy, it is replaced by an implemented specific to the distribution.

with something like

> Where possible, the generic code is replaced by explicit formulas or more accurate or efficient implementations.  For example, most distributions have an explicit formula for the cdf, so it is used instead of integrating the pdf.

Reply:
thanks, that reads nicer